larray/core/array.py

```diff
@@ -1380,7 +1380,8 @@ def reindex(self, axes_to_reindex=None, new_axis=None, fill_value=np.nan, inplac
         new_axis : int, str, list/tuple of str or Axis, optional
             List of new labels or new axis if `axes_to_replace` contains a single axis reference.
         fill_value : scalar or LArray, optional
-            Value set to data corresponding to added labels. Defaults to NaN.
+            Value used to fill cells corresponding to label combinations which were not present before reindexing.
+            Defaults to NumPy NaN.
```

Comment:
I wouldn't mention numpy, as the result would be the same if we used float('nan'). "Defaults to NaN" seems enough.